src/azure-cli-core/azure/cli/core/_session.py

```diff
@@ -75,8 +75,7 @@ def __len__(self):
         return len(self.data)
 
 # ACCOUNT contains subscriptions information
-# this file will be shared with azure-xplat-cli, which assumes ascii
-ACCOUNT = Session('ascii')
```

Comment:
azureProfile.json is being share with xplat.
Using `ascii` has been a workaround on windows for xplat because of the BOM, but not necessary now for reasons:
1. xplat has been fixed 3 releases ago to accept encoding of "utf with bom",
2. 'az' requires to work with the latest xplat anyway.
3. `ascii' always make me worry that the persistence will be broken someday if the `name` fields might contain non-ascii characters.